dkan_harvest.module

```diff
@@ -1,236 +1,677 @@
 <?php
-
 /**
  * @file
- * Harvesting module for dkan.
+ * Code for the DKAN Harvest feature.
  */
 
+include_once 'dkan_harvest.features.inc';
+
+// TODO Probably move this to drupal variables.
 define(
   'DKAN_HARVEST_CACHE_DIR',
   'public://dkan-harvest-cache'
 );
 
+define(
+  'DKAN_HARVEST_MIGRATION_PREFIX',
+  'dkan_harvest_migrate_'
+);
+
 /**
- * Returns configuration for harvest sources.
+ *
  */
-function dkan_harvest_sources_definition() {
-  $modules = module_implements('harvest_sources');
-  $subscribed_values = array();
-  // Get a list of licenses to display.
-  foreach ($modules as $module) {
-    $function = $module . '_harvest_sources';
-    $values = $function();
-    foreach ($values as $key => $value) {
-      // Make sure other module didn't provide the same source.
-      if (!isset($subscribed_values[$key])) {
-        $subscribed_values[$key] = $value;
-      }
-    }
+function dkan_harvest_menu() {
+  $items = array();
+
+  $items['node/%node/events'] = array(
+    'title' => 'Event Log',
```

Comment:
I'm not sure "event log" is the clearest thing to call this. I would call it "Harvest Log". Let's check with @federicca